libs/datastructures/include/hpx/datastructures/tuple.hpp

```diff
@@ -1017,4 +1092,10 @@ namespace hpx { namespace util {
 #pragma warning(pop)
 #endif
 
+#if defined(__clang__)
+#  pragma clang diagnostic pop
+#elif defined (__GNUC__)
+#  pragma GCC diagnostic pop
+#endif
```

Comment:
This `GCC diagnostic pop` has no corresponding `push`

Reply:
Good catch...